ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/openssl3/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+##################################################################
+# Install OpenSSL 3.0.10.1000                                    #
+# An OpenSSL version greater than 1.1.x is needed for DNF        #
+# See https://github.com/adoptium/infrastructure/issues/3274
+##################################################################
+---
+- name: Check if openssl is installed
+  stat:
+    path: /usr/bin/openssl
+  register: openssl_installed
+
+- name: Get version of installed openssl (if installed else skip)
+  shell: /usr/bin/openssl version | awk '{print$2}' | awk -F. '{print$1}'
+  register: openssl_version
+  when: openssl_installed.stat.exists
+
+- name: Install openssl if not installed or version is less than 3
+  when: (not openssl_installed.stat.exists) or (openssl_version.stdout_lines | int) < 3
+  block:
+    - name: Transfer openssl binary
+      unarchive:
+        src: /Vendor_Files/aix/openssl-3.0.10.1000.tar.Z
+        dest: /tmp/
+        remote_src: false
+
+    - name: cd into openssl directory and installp the binary
```

Comment:
What is it that fails without this PATH update? This feels a little indicative of a problem elsewhere?
Should probably have a comment about this for our future selves to be aware of the reason

Reply:
It doesn't necessarily fail if the PATH is not adjusted this way, but it will use `/opt/freeware/bin/uname` somewhere in the script instead of `/usr/bin/uname` which causes alot of 
```
uname: invalid option -- 'W'
Try 'uname --help' for more information.
```
in the output. Again this isn't causing the install to fail but i'd rather it use the `uname` it wants to use.

Ditto with the `shell: PATH=/usr/bin/:$PATH && /usr/sbin/updtvpkg` command